_ext.py

```diff
@@ -182,8 +189,8 @@ def sort_key(game):
         ]
         if len(combined_clones) > 0:
             combined['clones'] = combined_clones
-        parse_items(site, combined, 'remakes')
-        parse_items(site, combined, 'clones')
+        combined['both'] = combined_remakes + combined_clones
```

Comment:
use something like 'games' instead of 'both' - after all, one day this would include more things? So 'both' would be inaccurate 😉